aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +59,64 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
+
 ## When not to use
 
-- Do not use a Card to filter data on the same screen, instead use **\_**.
+- Do not use a Card as a clickable region to filter data on the same screen.
+- Do not use a Card to group complex content into a single container.
+- When data attributes across items needs to be analyzed, sorted, compared and filtered, a Card may not be the best tool. Instead, [DataTable](/components/datatable) may be more appropriate.
+
+<BestPracticeGroup>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'do',
+      message: 'Style filter buttons as default buttons.',
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardFilteringBestPractice />
+  </Example>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'dont',
+      message: `Don’t use a Card to wrap a set of buttons. Cards should act as independent regions that do not affect surrounding content.`,
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardFilteringBestPractice bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+<BestPracticeGroup>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'do',
+      message: 'Place non-Card page content flush on the page.',
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardContentBestPractice />
+  </Example>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'dont',
+      message: `Don’t use a Card solely as a way to denote a layout region of related content.`,
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardContentBestPractice bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
 
 ## Types of cards
 
```

Comment:
This section could use a brief intro, which speaks to a variety of card types, each with a primary purpose its solves for a user. A list of types with hyperlinks could be useful here.

Also, I think it might be nice to create headings for the following types with the primary purpose and "coming soon" messaging nested under each:
- Status-based navigation - Quickly assist the user to navigate to resources, tasks, or processes requiring their attention and/or action.
- Resource cards - Allow a user to scan available resource options and aid their choice based on their needs.
- Analytic cards - Present measures and metrics in short, digestible content pieces and allow a user to navigate to underlying detail is needed.

Reply:
I like that. Will add.

Reply:
Done.